processor/src/test/kotlin/com/tobrun/datacompat/DataCompatProcessorTest.kt

```diff
@@ -28,7 +28,7 @@ interface EmptyInterface2
 @Deprecated
 @DataCompat
 private data class PersonData(
-    @Default("\"John\"")
+    @Default("\"John\"", ["java.util.Date"])
```

Comment:
should `java.util.Date` be here? 
I don't use it, detekt will fail

Reply:
for tests it does not matter I guess - but yeah, let's use it to be consistent

Reply:
interesting why detekt didn't catch it

Reply:
cause those files are generated and used in runtime for tests

Reply:
this file not in `build/*` folder. so it's not generated. maybe copied